lib/deps.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+const CWD = process.cwd();
+
 const wt = require('node:worker_threads');
+const { createRequire } = require('node:module');
 const metautil = require('metautil');
+const appRequire = createRequire(`file://${CWD}/server.js`);
```

Comment:
What if the user starts app from some nested folder? Do we perhaps want to try "search up the tree" to find package.json?

Reply:
This solution expected  all metarhia projects to be of the same structure and they should have server.js in app root folder and I am ok with this